frontend/src/scenes/organization/Invites/CreateOrgInviteModal.tsx

```diff
@@ -53,11 +55,11 @@ export function CreateOrgInviteModalWithButton(): JSX.Element {
                 visible={isVisible}
             >
                 <p>
-                    Create an invite link for a teammate with a specific email address.
+                    Create an invite for a teammate with a specific email address.
                     <br />
-                    Remember to send the link to the teammate.
-                    <br />
-                    <i>Invites emailed by PostHog coming soon.</i>
+                    {user?.are_invite_emails_available
+                        ? 'PostHog will then email it to the address.'
+                        : "Since this PostHog instance isn't configured for emailing invites, remember to share the link!"}
                 </p>
                 <Input
```

Comment:
Per the new design guidelines, use a label instead of the add-on and I also suggest adding a placeholder with an example email (maybe even using the domain from the current user? GQ)

Reply:
As an improvement, submitting the form with the enter key would be great.